### DISCUSSION TITLE
Invio della posta elettronica tramite PHP

### DISCUSSION THREAD

**Discussion by giadaschiavone** [iisgiua/giuaschool#536] [Richieste di aiuto] [answered]
Buonasera.
Vorrei chiedervi aiuto, riguardante l'invio della posta elettronica tramite PHP. Nel dettaglio, questa sera stiamo configurando il registro elettronico per l'anno scolastico successiva e ci risulta, che configurando la posta elettronica tramite PHP ed inserendo l'indirizzo per inviare l'email di prova, l'email non arriva. Questo accade anche per le credenziali da inviare ai docenti o ai genitori. Nonostante ciò, ci risulta che è stato correttamente inviato.
Come possiamo risolvere la situazione? Fatemi sapere.

**Comment by trinko** [comment, marked as answer]
Effettuando l'invio della mail di prova, se non viene visualizzato nessun messaggio di errore e la mail non arriva, allora viene bloccata dal servizio di hosting.

Se usi **Altervista**, l'unico metodo funzionante è con il tipo: **"Usa server locale tramite PHP"**.

Se usi un altro hosting devi chiedere al tuo provider se c'è un blocco sull'invio delle email.

**Comment by giadaschiavone** [comment]
Anch'io ho fatto così! Ho usato il PHP, ma continua a non funzionare e/o ricevere niente. Inoltre, dice che nessun tipo di problema. 
Se dici che gli indirizzi e-mail sono il problema, gli abbiamo inserito correttamente e pure verificati due/tre volte.

**Comment by trinko** [comment]
Allora è probabile che l'errore sia dovuto all'indirizzo del mittente della mail: se questo non è un indirizzo valido, l'email non viene inviata dal sistema di posta.

Tutte le mail inviate dal registro hanno come mittente l'indirizzo che viene configurato in **SISTEMA > ISTITUTO** nel campo _"Indirizzo email del mittente delle notifiche inviate dal registro"_
Controlla che questo indirizzo sia configurato correttamente.

**Reply by giadaschiavone** [reply]
D'accordo, nei giorni successivi vedrò.
